scripts/gha/it_workflow.py

```diff
@@ -136,12 +140,16 @@ def test_start(token, issue_number, actor, commit, run_id):
 def test_progress(token, issue_number, actor, commit, run_id):
   """In PR, when some test failed, update failure info and 
   add label \"tests: failed\""""
-  log_summary = _get_summary_talbe(token, run_id)
-  if log_summary == 0:
+  success_or_only_flakiness, log_summary = _get_summary_talbe(token, run_id)
+  if success_or_only_flakiness and not log_summary:
     return
   else:
-    github.add_label(token, issue_number, _LABEL_FAILED)
-    comment = (_COMMENT_TITLE_PROGESS_FAIL +
+    if success_or_only_flakiness:
+      title = _COMMENT_TITLE_PROGESS_FLAKY
```

Comment:
Does this appaned a flaky title even on success?

Reply:
No. Added comment `all faliures/errors are due to flakiness (succeeded after retry)`